src/pages/contributing/translation-program/contributors.tsx

```diff
@@ -97,56 +81,69 @@ const Contributors = ({
 
       <Content>
         <Breadcrumbs slug={location.pathname} />
-        <h1>
+        <ContentHeading as="h1">
           <Translation id="page-contributing-translation-program-contributors-title" />
-        </h1>
-        <h4>
-          <strong>
+        </ContentHeading>
+        <ContentHeading as="h4">
+          <Text as="strong">
             <Translation id="page-contributing-translation-program-contributors-number-of-contributors" />{" "}
             {translatorData.length}
-          </strong>
-        </h4>
-        <p>
+          </Text>
+        </ContentHeading>
+        <Text>
           <Translation id="page-contributing-translation-program-contributors-our-translators-1" />
-        </p>
-        <p>
+        </Text>
+        <Text>
           <Translation id="page-contributing-translation-program-contributors-our-translators-2" />
-        </p>
-        <p>
+        </Text>
+        <Text>
           <Translation id="page-contributing-translation-program-contributors-our-translators-3" />
-        </p>
-        <p>
+        </Text>
+        <Text>
           <Translation id="page-languages-interested" />{" "}
           <Link to="/contributing/translation-program/">
             <Translation id="page-languages-learn-more" />
           </Link>
           .
-        </p>
-        <h2>
+        </Text>
+        <ContentHeading as="h2">
           <Translation id="page-contributing-translation-program-contributors-thank-you" />
-        </h2>
-        <HorizontalUl>
+        </ContentHeading>
+        <UnorderedList
+          display="grid"
+          gridTemplateColumns={{
+            base: "1fr",
+            sm: "repeat(2, 1fr)",
+            md: "repeat(3, 1fr)",
+            lg: "repeat(4, 1fr)",
+            xl: "repeat(6, 1fr)",
+          }}
+        >
```

Comment:
We can utilize the `as` prop here to combine two Chakra Props. I like using `SimpleGrid` here to give you some shortcut props to simplify the readability. And because we are changing the number of columns through many breakpoints from the smallest with no skips, we can use array bracketing to simplify the syntax further.

Also, `UnorderedList` by default renders `margin-inline-start` with its own value which adds greater specificity then `margin-left`. This is overriding a `margin-left` value of `1.45rem`  from global styles with `1em`. 

```suggestion
        <SimpleGrid as={UnorderedList} columns={[ 1, 2, 3, 4, 6 ]} ms="1.45rem">
```

Reply:
Interesting approach! much cleaner, thanks for sharing this.